05/DB.test.js

```diff
@@ -1 +1,9 @@
-import DB from './DB';
\ No newline at end of file
+import DB from "./DB";
+
+it("when data id is a number insert to database  ", async () => {
+  const database = new DB();
+
+  const promise = await database.insert({ id: 1 });
```

Comment:
Jeśli używasz `await` to do zmiennej przypisujesz wartość jaką zwróci obietnica czyli w Twoim przypadku `promise` to tak naprawdę `valueFromPromise` -> `.then(valueFromPromise => ...)`.

Wszystkie metody w DB są asynchroniczne więc musisz obsługiwać je w ten sposób. W dokumentacji znajdziesz informacje o tym jakie masz opcje: https://jestjs.io/docs/asynchronous.

Jeśli masz problem ze zrozumieniem co zwraca dana metoda to wyświetl te dane w konsoli.
Jak uruchomisz test to wynik tego console.loga zobaczysz powyżej statystyk dot. testów.

Daj znać na Slacku czy taka podpowiedź wystarczy :)